zebra-script/src/lib.rs

```diff
@@ -119,26 +121,28 @@ impl CachedFfiTransaction {
         let script_len = script_pub_key.len();
 
         let amount = value.into();
+
         let flags = zcash_script::zcash_script_SCRIPT_FLAGS_VERIFY_P2SH
             | zcash_script::zcash_script_SCRIPT_FLAGS_VERIFY_CHECKLOCKTIMEVERIFY;
+        #[cfg(windows)]
+        let flags = flags
+            .try_into()
+            .expect("zcash_script_SCRIPT_FLAGS_VERIFY_* enum values fit in a c_uint");
 
         let consensus_branch_id = branch_id.into();
 
         let mut err = 0;
 
+        // SAFETY: `new` makes sure `self.precomputed` is not NULL
```

Comment:
Nit: Tweak to make it a bit clearer.

```suggestion
        // SAFETY: `Self::new` makes sure `self.precomputed` is not NULL
```

Reply:
`Self` isn't searchable, and you need to look outside the current method to fully understand it. That's a particular problem for unsafe code, where we want a complete SAFETY explanation right next to the code.

So I used the full type name instead.

See e6cd5b87d